app/src/main/java/org/wikipedia/travel/database/TripDbHelper.java

```diff
@@ -114,6 +117,58 @@ public Trip getFullListById(long id) {
         return list;
     }
 
+    public Void addUserLandmarks(long tripId, List<LandmarkCard> landmarks) {
```

Comment:
Is the reason as to why it's `Void` vs `void` because of the CallbackTask?

Reply:
Yeah exactly. Previously I set it as Object, but i noticed the devs using Void instead. I guess it's more comprehensive.